UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterPlayer.cs

```diff
@@ -219,6 +220,11 @@ public override void MatrixChange(Matrix MatrixOld, Matrix MatrixNew)
 		{
 			MatrixOld.PresentPlayers.Remove(this);
 			MatrixOld.UpdatedPlayerFrame = Time.frameCount;
+			AdminLogsManager.AddNewLog(
+				gameObject,
+				$"{playerScript.playerName} has left {MatrixOld.name}",
+				LogCategory.World
```

Comment:
you can properly add to where they go to, though this might be a bit Spamming

Reply:
nu uh